src/olympia/abuse/migrations/0027_add_cinderdecision.py

```diff
@@ -0,0 +1,63 @@
+# Generated by Django 4.2.10 on 2024-03-11 17:03
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import olympia.amo.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        # migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        # ('bandwagon', '0008_alter_collection_options_alter_collection_managers_and_more'),
+        # ('addons', '0049_clear_bad_url_data'),
+        # ('ratings', '0011_remove_rating_one_review_per_user_and_more'),
```

Comment:
the migration failed locallywhen I ran it, seemingly because of one of those migrations, and I forgot to go back to investigate _where_ it was failing.  I'll poke around.  Really, the 0001 of any of them should be enough - we only need the pk.

Reply:
Worked fine this time so must have been an unrelated problem (or something I fixed along the way)
https://github.com/mozilla/addons-server/pull/22023/commits/d0ea08851d8dc3eb7919c27c5f8ca38c5dea5fee